ddtrace/appsec/__init__.py

```diff
@@ -0,0 +1,38 @@
+from ddtrace.internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+_appsec_import_error = None
+
+try:
+    from ddtrace.appsec.processor import AppSecProcessor
+except ImportError as e:
+    from ddtrace import config
+
+    if config._raise:
+        raise
+
+    AppSecProcessor = None  # type: ignore
+    _appsec_import_error = repr(e)
+
+
+def enable(*args, **kwargs):
+    if AppSecProcessor is None:
+        # DDAS-001-01
```

Comment:
Should we add these to the log messages themselves? Really like this idea in general though, I'd love to do this for all our warning/error level logs